### ISSUE TITLE
A11y_Azure Tools Storage Explorer_ Add a Blob Container_ Attach with Azure AD_Screenreader: Screenreader is not announcing the name of the " Select Account & Tenant" dialog box when it gets open.

### ISSUE THREAD

**Issue by Jtith** [microsoft/AzureStorageExplorer#6325]
**GitHub Tags:**

#A11yMAS;#A11yTCS;#Win11;#DesktopApp;#A11ySev2;#BM_AzureToolsStorageExplorer_Win32_Dec2022;#Azure Tools Storage Explorer;#FTP;#Screenreader;#WCAG1.3.1;#NVDA;#Element:Dialog;#Win32;#Info and relationship; 

## Actual results:
Screenreader is not announcing the name of the " Select Account & Tenant" dialog box when it gets open.

**Issue is not observed with JAWS**

**Similar issue also observed in below scenario screens: -**

1. Open connect dialog and tab to ADLS Gen2 container or directory- Select Connect Method, select the "Sign in using Azure Active Directory (Azure AD)" radio button. NVDA is not announcing the **"Select Account & Tenant" dialog**
2. Open connect dialog, tab to Storage account or service - "Select Connect Method", select the "Sign in using Azure Active Directory (Azure AD)" radio button then Next. NVDA is not announcing the **"Enter Connection Info" dialog**
3. Open connect dialog, tab to Storage account or service -In Select Connection Method, select radio button of  "Connection string (Key or SAS)", then Next. NVDA is not announcing the **"Enter Connection Info" dialog**

## Expected results:
Screen reader should announce the name of the " Select Account & Tenant" dialog box.
## Environment Details:
Application: Microsoft Azure Storage Explore
version: 1.27.0
NVDA version 2020.4
JAWS version 2021.205.53
## OS Details
Microsoft Windows 11 Enterprise
version 22H2 OS Build 22621.819
## Repro Steps:

1. Launch Storage Explorer
4. Tab to "Open Connect Dialog" pane present on left side of screen and hit enter. 
5. In the "Connect to Azure Storage" dialog, select "Blob container", then hit Next.
6. In "Select Connect Method" will open 
7. Turn on NVDA
8. And select the "Sign in using Azure Active Directory (Azure AD)" radio button. Then press Next button
9. You will observe that screen reader is not announcing the name of the " Select Account & Tenant" dialog box when it gets open.

## User Impact:
Screen Reader users will not know the dialog box is opened.

## Attachments:

https://user-images.githubusercontent.com/100552367/205280485-37ab1bc9-ccbb-4b24-bdba-4ad048276d89.mp4

https://user-images.githubusercontent.com/100552367/205302919-2b831bf0-79f4-4e3c-86e6-2a392e74ef39.mp4

**Comment by Jtith** [comment]
issue is fixed in the latest build 1.28.0, hence closing the bug
![6325_issue fixed](https://user-images.githubusercontent.com/100552367/211462151-8cd94f27-4c70-4ae3-a95d-7a0badea990a.png)

**Comment by tripathiakhil429** [comment]
#Closed
